pkg/nmi/server/server.go

```diff
@@ -17,15 +17,15 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/Azure/go-autorest/autorest/adal"
```

Comment:
nit: exporting external libraries should be placed after exporting internal libraries.